keras_cv/layers/preprocessing/mix_up_test.py

```diff
@@ -32,25 +32,35 @@ def test_return_shapes(self):
             "classes": tf.random.uniform((2, 3), 0, 1),
         }
 
+        # randomly sample segmentation mask
+        ys_segmentation_masks = tf.random.uniform(
+            shape=(2, 512, 512), maxval=10
+        )
+        ys_segmentation_masks = tf.cast(ys_segmentation_masks, tf.uint8)
+        ys_segmentation_masks = tf.one_hot(ys_segmentation_masks, num_classes)
+
         layer = MixUp()
         # mixup on labels
         outputs = layer(
             {
                 "images": xs,
                 "labels": ys_labels,
                 "bounding_boxes": ys_bounding_boxes,
+                "segmentation_masks": ys_segmentation_masks,
             }
         )
-        xs, ys_labels, ys_bounding_boxes = (
+        xs, ys_labels, ys_bounding_boxes, ys_segmentation_masks = (
             outputs["images"],
             outputs["labels"],
             outputs["bounding_boxes"],
+            outputs["segmentation_masks"],
         )
 
         self.assertEqual(xs.shape, [2, 512, 512, 3])
         self.assertEqual(ys_labels.shape, [2, 10])
         self.assertEqual(ys_bounding_boxes["boxes"].shape, [2, 6, 4])
         self.assertEqual(ys_bounding_boxes["classes"].shape, [2, 6])
+        self.assertEqual(ys_segmentation_masks.shape, [2, 512, 512, 10])
 
     def test_mix_up_call_results(self):
```

Comment:
Let's add a `segmentation_mask` to the test data in this test and make the same assertion (that the resulting masks are, in fact, different)

Reply:
`ys_segmentation_masks = tf.random.uniform(shape=(2, 4, 4), maxval=10)` is going to give you uniform between zero and 1, so the resulting masks after casting to unit8 may not be what you're expecting. Probably we can just use the same `ones` and `2 * ones` approach for the segmentation masks.

Reply:
Ooh true. Fixed it.